package.json

```diff
@@ -42,7 +42,9 @@
     "auto:lerna:build": "lerna run build",
     "auto:ts:build": "tsc --build",
     "auto:ts:build:watch": "tsc --build --watch",
-    "auto:ts:references": "node setup-references.mjs"
+    "auto:ts:references": "node setup-references.mjs",
+    "prettier": "prettier --write \"*/**/*.{ts,tsx,js,jsx,json,css,scss,md}\"",
+    "prepare": "husky install"
```

Comment:
pls describe in readme this step or maybe even better to keep consistent with DN FE repo 

Readme.md:
`Run `npx husky-init` after you first time clone project`

Reply:
This is not required, it will work without additional initialization